sdk/identity/identity/src/credentials/managedIdentityCredential/index.ts

```diff
@@ -19,6 +19,16 @@ import { fabricMsi } from "./fabricMsi";
 
 const logger = credentialLogger("ManagedIdentityCredential");
 
+/**
+ * Options to send on the {@link ManagedIdentityCredential} constructor.
+ */
+export interface ManagedIdentityCredentialOptions extends TokenCredentialOptions {
+  /**
+   * Allows specifying a custom resource Id.
+   */
+  resourceIdentifier?: string;
+}
```

Comment:
Why not put both clientId and resourceIdentifier in this options bag? Do users have to set one through an explicit param but the other in the options?

Reply:
I’ll add it! — the right way to do it feels like a breaking change, but I can make an additive change, so I’ll do that 🙂